flake.lock

```diff
@@ -410,11 +410,11 @@
     },
     "nixpkgs_2": {
       "locked": {
-        "lastModified": 1726151616,
-        "narHash": "sha256-Pu+odXzikSAkl/39zdvB86hasDrEImixpU3RXi45I74=",
+        "lastModified": 1726214531,
+        "narHash": "sha256-6sXBTZ0R5t1oRK57vFSs5TyClHmID9Ih8tlQvrVf/0M=",
         "owner": "flyingcircusio",
         "repo": "nixpkgs",
-        "rev": "6ce38e4d52cbb7313f5170903deb4b8b3d3ff022",
+        "rev": "4c934f9fa9a0c9239a1bc2817d01231807f5d2ba",
```

Comment:
Given that there's no corresponding change in the flake.nix, I suppose you just used `--override-input` as a one-off action?

Reply:
just a normal nixpkgs update with push to nixos-24.05